src/main/java/de/tum/in/www1/artemis/repository/ComplaintRepository.java

```diff
@@ -86,16 +88,63 @@
     long countUnacceptedComplaintsByComplaintTypeTeamShortNameAndCourseId(@Param("teamShortName") String teamShortName, @Param("courseId") Long courseId);
 
     /**
-     * This magic method counts the number of complaints by complaint type associated to an exercise id
+     * This method counts the number of complaints by complaint type associated to an exercise id
      *
      * @param exerciseId    - the id of the course we want to filter by
      * @param complaintType - complaint type we want to filter by
      * @return number of complaints associated to exercise exerciseId
      */
-    long countByResult_Participation_Exercise_IdAndComplaintType(Long exerciseId, ComplaintType complaintType);
+    @Query("""
+                SELECT count(c) FROM Complaint c
```

Comment:
Style nit here and below: If we want to write all queries upper case, use `COUNT`.

Reply:
I capitalized it for consistnecy